app/controllers/skins_controller.rb

```diff
@@ -120,9 +114,10 @@ def update
   def preview
     flash[:notice] = []
     flash[:notice] << ts("You are previewing the skin %{title}. This is a randomly chosen page.", title: @skin.title)
-    flash[:notice] << ts("Go back or click any link to remove the skin.")
-    flash[:notice] << ts("Tip: You can preview any archive page you want by tacking on '?site_skin=[skin_id]' like you can see in the url above.")
-    flash[:notice] << "<a href='#{skin_path(@skin)}' class='action' role='button'>".html_safe + ts("Return To Skin To Use") + "</a>".html_safe
+    flash[:notice] << ts("Go back or follow any link to remove the skin.")
+    flash[:notice] << (ts("Tip: You can preview any archive page you want by adding '?site_skin=%{skin_id}' to the end of the URL.", skin_id: @skin.id) +
+                      "<p><a href='#{skin_path(@skin)}' class='action'>".html_safe + ts("Return to Skin to Use") + "</a></p>".html_safe)
```

Comment:
Same thing about the translation helper, but also noting that this does not follow our current [internationalisation standards](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards). Generally, we'd now prefer something like
```suggestion
    flash[:notice] << t(".preview_tip_html",
      skin_id: @skin.id,
      return_to_skin_link: link_to(t(".return_to_skin"), skin_path(@skin)))
```
with the corresponding keys in https://github.com/otwcode/otwarchive/blob/master/config/locales/controllers/en.yml. (I haven't run the above through a linter, so some tweaks will probably be needed for it to work perfectly.)

Additionally, make sure to swap out the single quotes around `'site_skin?=%{skin_id}'` to be double quotes `"site_skin?=%{skin_id}"` (this should require less escaping when things are moved into the YAML file, so I'd recommend doing that move first).